Code/Source/solver/vtk_xml_parser.cpp

```diff
@@ -466,7 +466,7 @@ void store_element_conn(vtkSmartPointer<vtkUnstructuredGrid> vtk_ugrid, mshType&
 void store_element_ids(vtkSmartPointer<vtkUnstructuredGrid> vtk_ugrid, mshType& mesh)
 {
   auto elem_ids = vtkIntArray::SafeDownCast(vtk_ugrid->GetCellData()->GetArray(ELEMENT_IDS_NAME.c_str()));
-  if (elem_ids == nullptr) { 
+  if (elem_ids == nullptr && !mesh.vrtual) { 
```

Comment:
This is why if we treat a virtal face as not actually used like other faces we have to add all of these checks.